src/shared/tmpfile-util-label.c

```diff
@@ -24,3 +24,21 @@ int fopen_temporary_label(
 
         return r;
 }
+
+int open_tmpfile_linkable_label(
+                const char *target,
+                int flags,
+                char **ret_path) {
```

Comment:
hmm, do we really want individual wrappers around all these helper?

I'd prefer if we went down this route instead:
https://github.com/systemd/systemd/pull/20767#discussion_r714732194